Extensions/PrimitiveDrawing/pixi-graphics-extras/graphics-extras.js

```diff
@@ -0,0 +1,406 @@
+/*!
```

Comment:
Maybe we should use the [minified version](https://unpkg.com/browse/@pixi/graphics-extras@6.0.0/dist/browser/graphics-extras.min.js)?